Dockerfile-es

```diff
@@ -0,0 +1,3 @@
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.4.3
```

Comment:
Can you refresh my memory: is this the version we're using in AWS-land? Or maybe it doesn't matter to keep these in lockstep?

Reply:
Dang, just a second too late! :laughing:

Reply:
We are using 6.4 in AWS-land.

Reply:
:tada: thanks, @justinlittman